src/Resources/app/administration/src/module/nosto/components/nosto-integration-account-general/nosto-integration-account-general.html.twig

```diff
@@ -86,16 +86,6 @@
                     name="searchToken"
                 />
             {% endblock %}
-
-            {% block nosto_listing_sending %}
```

Comment:
The button was initially implemented for CM1, which was removed. It could be useful for some search indexing in the future (suggested by Olsi), so I don't want to remove all the code behind it right now.